src/components/forms/FormEditor.tsx

```diff
@@ -97,6 +100,24 @@ const FormEditor = ({ form }: FormEditorProps) => {
     );
   };
 
+  const onCreateFormTemplate = async () => {
+    const newForm: FormCreate = {
+      title: templateTitle, // her kommer det en popup der man kan skrive inn navnet til templaten
+      fields: fields,
+      resource_type: FormResourceType.FORM,
+      template: true,
+    };
+    createForm.mutate(newForm, {
+      onSuccess: (data) => {
+        showSnackbar(data.title + ' malen ble lagret', 'success');
+      },
+      onError: (e) => {
+        showSnackbar(e.detail, 'error');
+      },
+    });
+    setFormDialogOpen(false);
```

Comment:
```suggestion
    createForm.mutate(newForm, {
      onSuccess: (data) => {
        showSnackbar(data.title + ' malen ble lagret', 'success');
        setFormDialogOpen(false);
      },
      onError: (e) => {
        showSnackbar(e.detail, 'error');
      },
    });
```

Det er bedre å lukke dialogen kun hvis det går bra. Hvis det går dårlig er det fint å la dialogen bli stående oppe.